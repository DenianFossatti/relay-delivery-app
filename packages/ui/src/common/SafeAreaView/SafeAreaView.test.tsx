import { mockTestID, render, wrapper } from '../../tests';

import { SafeAreaView, SafeAreaViewProps } from '.';

describe(`SafeAreaView`, () => {
  const props: SafeAreaViewProps = {
    testID: 'SafeAreaView',
  };

  const el_container = mockTestID('View', props.testID!);

  it(`render component #${el_container}`, async () => {
    const { findByTestId, toJSON } = render(<SafeAreaView {...props} />, {
      wrapper,
    });
    const sut = await findByTestId(el_container);
    expect(sut).toBeTruthy();
    expect(toJSON()).toMatchSnapshot();
  });
});
