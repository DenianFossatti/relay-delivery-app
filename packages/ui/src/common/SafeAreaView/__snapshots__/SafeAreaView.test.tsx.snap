// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms/SafeAreaView render component #View_SafeAreaView 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <RNCSafeAreaView
    accessible={false}
    edges={
      [
        "bottom",
        "left",
        "right",
        "top",
      ]
    }
    style={
      {
        "backgroundColor": "#fff",
        "flex": 1,
      }
    }
    testID="View_SafeAreaView"
  />
</RNCSafeAreaProvider>
`;
